src/Mobs/Monster.cpp

```diff
@@ -1129,7 +1135,6 @@ bool cMonster::WouldBurnAt(Vector3d a_Location, cChunk & a_Chunk)
 		(a_Chunk.GetSkyLight(RelX, RelY, RelZ) == 15) &&             // In the daylight
```

Comment:
Unrelated, but the usage of chunks are causing issues, since I believe mobs can move to a different chunk (invalidating the parameter) in super::Tick, and then this function is called.

Fix is to use the GET_AND_VERIFY_CURRENT_CHUNK macro and place it under super::Tick in this file, I think.